pyTST/core.py

```diff
@@ -215,35 +243,68 @@ def onclick(event):
 
                 index = min(np.searchsorted(self.step_time_array, event.xdata), len(self.step_time_array) - 1)
                 update_cursor(index)
+
                 pyplot.draw()
+                global indexGlobal 
```

Comment:
is a global variable *really* needed. These should be avoided, and are 90% of the time not useful. Here you can just use self.current_index and update that in the onclick function I think

Reply:
(and again camel case when it should be snake case

Reply:
In principle I completely agree! However I couldn't get it to work without it (I spend quite some time on it yesterday), but I'll try your suggestion